mips/intr.c

```diff
@@ -169,6 +169,23 @@ static void fpe_handler(exc_frame_t *frame) {
   }
 }
 
+static void cp_unusable_handler(exc_frame_t *frame) {
+  int cp_id = (frame->cause & CR_CEMASK) >> CR_CESHIFT;
+  bool kernel_mode = (frame->sr & SR_KSU_MASK) == 0;
+
+  if (cp_id != 1) {
+    panic(
```

Comment:
> A COP0 or Cache instruction was executed while the processor was running in a mode other than Debug Mode
or Kernel Mode, and the CU0 bit in the Status register was a zero.

Every userspace process executing COP0 instruction causes kernel to crash.
Maybe we should kill a userspace process which caused that instead calling panic?

Reply:
An attempt to execute `COP0` instruction in user-space should end up by sending `SIGILL` to the offending processes. However I believe illegal instruction (aka reserved) should be services by `EXC_RI` handler.